app/src/main/java/de/danoeh/antennapod/activity/WidgetConfigActivity.java

```diff
@@ -54,7 +64,10 @@ protected void onCreate(Bundle savedInstanceState) {
         displayDeviceBackground();
         opacityTextView = findViewById(R.id.widget_opacity_textView);
         opacitySeekBar = findViewById(R.id.widget_opacity_seekBar);
-        widgetPreview = findViewById(R.id.widgetLayout);
+        previewLayout = findViewById(R.id.widget_config_preview);
+        widgetPreview = previewLayout.findViewById(R.id.widgetLayout);
+        widgetMorePreview = findViewById(R.id.widgetMoreLayout);
+        previewMoreLayout = findViewById(R.id.widget_config_more_preview);
```

Comment:
I'm not completely happy about the name `More`, to be honest. How about `Extended`?

Reply:
done